base-system/prepare.sh

```diff
@@ -59,6 +59,8 @@ ln -sf /etc/dpkg/origins/huronos /etc/dpkg/origins/default
 ln -sf /usr/lib/os-release /etc/os-release
 
 ## hsync symlinks
+chmod 744 /usr/lib/systemd/system/ipman.service
```

Comment:
Okay, added the chmod.
And I agree, although I'm thinking that once a PR is in review it cannot go to a "do not review this PR yet, I'm working on it rn" state, or can it?
It would be quite useful so I save you time on mid-baking PRs

Reply:
> Okay, added the chmod. And I agree, although I'm thinking that once a PR is in review it cannot go to a "do not review this PR yet, I'm working on it rn" state, or can it? It would be quite useful so I save you time on mid-baking PRs

I know, Github UI is confusing haha, I had the same problem. You can set it to re-set it to draft again in the reviewers section:
<img width="307" alt="Captura de pantalla 2023-08-04 a la(s) 8 58 51 a m" src="https://github.com/equetzal/huronOS-build-tools/assets/62947298/113b0354-bd1e-4555-b203-47e16ba2495c">
This is usually done when you'll be doing hard changes to the PR. When their are only light you usually send changes and re-ask for review when needed. It might be a good idea to later setup a bot for specifically asking for reviews and ignore the Github notifications about updates; most companies have something similar to this but I've not explored that for now. 
For now just feel free to setup to draft your PRs until you feel they are ready to be reviewed again! :D